tests/unit/regression/krrnystrom_unittest.cc

```diff
@@ -0,0 +1,186 @@
+/*
+* Copyright (c) The Shogun Machine Learning Toolbox
+* Written (w) 2016 Fredrik Hallgren
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+*
+* 1. Redistributions of source code must retain the above copyright notice, this
+* list of conditions and the following disclaimer.
+* 2. Redistributions in binary form must reproduce the above copyright notice,
+* this list of conditions and the following disclaimer in the documentation
+* and/or other materials provided with the distribution.
+*
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+* ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+* WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+* DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+* ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+* (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+* LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+* ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+* (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+* SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*
+* The views and conclusions contained in the software and documentation are those
+* of the authors and should not be interpreted as representing official policies,
+* either expressed or implied, of the Shogun Development Team.
+*/
+
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/labels/RegressionLabels.h>
+#include <shogun/kernel/GaussianKernel.h>
+#include <shogun/regression/KRRNystrom.h>
+#include <shogun/regression/KernelRidgeRegression.h>
+#include <shogun/base/some.h>
+#include <gtest/gtest.h>
+
+#ifdef HAVE_CXX11
+
+using namespace shogun;
+
+/**
+ * Test setters are available
+ */
+TEST(KRRNystrom, test_setters)
+{
+	/* Use default constructor */
+	auto krr=some<CKRRNystrom>();
+
+	/* Test setters */
+	krr->set_tau(0.1);
+	krr->set_epsilon(0.1);
+	krr->set_num_rkhs_basis(100);
```

Comment:
if you really want to test those, you should call the getter afterwards and ensure it returns the same you set